trace-utils/src/send_data/mod.rs

```diff
@@ -458,6 +471,7 @@ mod tests {
     use httpmock::MockServer;
     use std::collections::HashMap;
 
+    const MILLIS_PER_SECOND: u64 = 1_000;
```

Comment:
I'm not sure this is the best description of the value (even though it is technically true). Maybe `TIMEOUT_MS` is more correct?